src/app/hooks/useSearch.ts

```diff
@@ -0,0 +1,26 @@
+import useFetch from './useFetch';
+import type { Gear } from '../../types';
+import { useEffect, useState } from 'react';
+
+export default function useSearch(searchValue: string): {
+  gear: Gear[] | null;
+  errorMessage: string | null;
+  isLoading: boolean;
+} {
```

Comment:
I think the `searchValue` state in `SearchPage.tsx` could live inside this hook as well. This would make this hook a little less versatile, but since you only use it to search gear based on a changeable value, you might as well have it bundled with the hook.

Reply:
Thanks for your review! 🙏
I think I understand, but I still would like to have a explanation, if possible @marwinburesch